REVIEWING.md

```diff
@@ -0,0 +1,25 @@
+# Review Guidelines
+
+This repository is shared between many teams, and it is not always clear what a reviewer should be looking for.
+The following guidelines are intended to help.
+This is a _work in progress_ and additions to this list are welcome.
+
+## Practicalities
+
+If any of the `.proto` files are changed, then the corresponding Go code should be regenerated in the same PR (`GOPATH=$(go env GOPATH) rake codegen`).
```

Comment:
Our code (golang) also contains tests. Usually we `codegen` the code as well as run the `tests` or both `rake` with default target.

Reply:
Oh, I wasn't aware that tests are run in the agent-payload ci. In case it does above sense 👍

Reply:
If they're not, that should be fixed!

Reply:
Oh, they're not!  I wrote #191 to do so.